bot/resources/tags/sql-fstring.md

```diff
@@ -12,5 +12,5 @@ db.execute(query, params)
 Note: Different database libraries support different placeholder styles, e.g. `%s` and `$1`. Consult your library's documentation for details.
 
 **See Also**
-• [Extended Example with SQLite](https://docs.python.org/3/library/sqlite3.html) (search for "Instead, use the DB-API's parameter substitution")
+• [Python sqlite3 docs](https://docs.python.org/3/library/sqlite3.html#how-to-use-placeholders-to-bind-values-in-sql-queries) - How to use placeholders to bind values in SQL queries
```

Comment:
Could you match the capitalization of SQLite to the old version.

Reply:
Note that `sqlite3` and SQLite is two different pieces of software:

- sqlite3 is the standard library extension module in CPython
- SQLite is the underlying C library used by the sqlite3 extension module

The link points to the Python documentation for the standard library extension module _sqlite3_, not to the SQLite documentation. The SQLite docs can be found here: https://www.sqlite.org/docs.html